.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: |
           sudo add-apt-repository multiverse
           sudo apt update
-          sudo apt install rar unrar
+          sudo apt install rar unrar libssl-dev libcurl4-openssl-dev zlib1g-dev
       - name: Setup Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
```

Comment:
In order to let `cmake` to find the right Python version, we may need to add right Python liib to `$PATH`

Reply:
Or, you can follow this instruction https://github.com/actions/setup-python/issues/186#issuecomment-1010828971

Reply:
> Or, you can follow this instruction [actions/setup-python#186 (comment)](https://github.com/actions/setup-python/issues/186#issuecomment-1010828971)

Added.